main.py

```diff
@@ -43,5 +46,19 @@ def main(input_code, input_lang, output_lang):
     
     output_code = gtc.gast_to_code(gast, output_lang)
 
+    # Use a service account
```

Comment:
Consider moving this to a separate function to keep the main.py function more readable.

Reply:
+1 can we move all firebase code to its own file?